cypress/integration/upload.spec.ts

```diff
@@ -17,7 +17,6 @@ describe("Upload from the the landing page", function () {
   it("should find a button to press to upload a file", function () {
     cy.visit("/");
     // we need a valid bcp47Tag to update the packageInfo
-    cy.contains("label", "Enter BCP 47 tag").scrollIntoView();
     cy.get("#tag-input").type("zh", { force: true });
```

Comment:
I think this should also invoke `disableSmoothScroll()` in `beforeEach` though...

```suggestion
    cy.get("#tag-input").type("zh");
```